.github/workflows/benchmark.yml

```diff
@@ -83,7 +83,8 @@ jobs:
           ASV_SKIP_SLOW: 1
         run: |
           set -x
-          python -m pip install asv virtualenv packaging
+          python -m pip install virtualenv packaging
+          python -m pip install -r requirements/benchmarking.txt
```

Comment:
Instead of doing this manually, can we just move `virtualenv` to the `benchmarking.txt` requirements? I don't think there's any harm in doing - in fact, it guarantees that anyone can run benchmarking instead of just `conda` users!

Also - is `packaging` necessary?

Reply:
> packaging

It used to be! I'm not sure if `asv` fixed the dependency issue with virtualenv and packaging. Need to check this.

Reply:
FWIW I never install it when I run benchmarks locally - but I always use asv+virtualenv, so maybe it is required when using conda? Either way NBD!